src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationSource.cs

```diff
@@ -29,22 +29,22 @@ public class DaprSecretStoreConfigurationSource : IConfigurationSource
         /// <summary>
         /// Gets or sets the custom key delimiters. Contains the '__' delimiter by default.
         /// </summary>
-        public IList<string> KeyDelimiters { get; set; } = new List<string> { "__" };
+        public IList<string>? KeyDelimiters { get; set; } = new List<string> { "__" };
```

Comment:
Would prefer that this not be made nullable

Reply:
Do you also want me to to change `DaprSecretStoreConfigurationProvider` to not allow a null value for this? Or only make it non nullable in this path?